contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java

```diff
@@ -120,9 +120,9 @@ public void run() throws Exception {
         }
 
         AdminClientConfig adminConfig = new AdminClientConfig().setMaxConnectionsPerNode(cluster.getNumberOfNodes())
-                                                               .setAdminConnectionTimeoutSec(15)
+                                                               .setAdminConnectionTimeoutSec(60)
                                                                .setMaxBackoffDelayMs(maxBackoffDelayMs)
-                                                               .setAdminSocketTimeoutSec(60);
+                                                               .setAdminSocketTimeoutSec(60 * 5); // Admin request timeout: 5min
```

Comment:
Can you leave a comment saying that HdfsFailedFetchLock could take long time when the Name Node is in a different data center. So expanding it to 5 minutes.